modules/troubleshooting-disabling-autoreboot-mco-cli.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+[id="troubleshooting-disabling-autoreboot-mco-cli_{context}"]
+= Disabling Machine Config Operator from automatically rebooting by using the CLI
+
+To avoid unwanted disruptions from changes made by the Machine Config Operator (MCO), you can use the CLI to modify the machine config pool (MCP) to prevent automatic rebooting after the Operator makes changes to the machine config.
+
+[NOTE]
+====
+Pausing a machine config pool stops all system reboot processes and all configuration changes from being applied.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+* You have root access in {product-title}.
```

Comment:
I'm not an expert on authorization in OpenShift but I haven't seen the term `root access` mentioned.  Is this a thing?  Most of what I've read deals with cluster roles and bindings.

Reply:
This was in the doc previously. A good question, though.
@bobfuru can you help here? What is root access in OpenShift? I've looked at a few other instances of `oc patch` in the docs and don't see the need to be root.

Reply:
IIRC, I added this because the original [BZ1886712](https://bugzilla.redhat.com/show_bug.cgi?id=1886712) asked to include [this KCS article](https://access.redhat.com/solutions/5477811), which shows root access used to run the commands. I couldn't find other discussion as to if or why this is required in this section, though. Also IIRC, the wording "root access" was used here because it also occurs in another Troubleshooting section: https://docs.openshift.com/container-platform/4.7/support/troubleshooting/investigating-pod-issues.html#starting-debug-pods-with-root-access_investigating-pod-issues

All that being said, I wonder if root access is needed at all here? @mburke5678, I know you added the console instructions after my initial PR; doesn't seem that would need root. I suggest removing this line here and in the console section, and changing `#` prompts to `$` prompts. 

@pneedle-rh Any thoughts from you?

Reply:
Using `oc debug` will give you root access in a running pod/container.  The actions describe in this doc are not related to root access and I think it can be dropped.   As Bob said, the prompts should be changed from `#` to `$`.